design/backup_v2_partitioned_logs.md

```diff
@@ -256,8 +257,16 @@ The command line for pause or resume backups remains the same, but the implement
 
 * Partitioned mutation logs are stored in `plogs/XXXX/XXXX` directory and their names are in the format of `log,[startVersion],[endVersion],[UID],[N-of-M],[blockSize]`, where `M` is total partition number, `N` can be any number from `0` to `M - 1`. In contrast, old mutation logs are stored in `logs/XXXX/XXXX` directory and are named differently.
 * To restore a version range, all partitioned logs for the range needs to be available. The restore process should read all partitioned logs, and combine mutations from different logs into one mutation stream, ordered by `(commit_version, subsequence)` pair. It is guaranteed that all mutations form a total order. Note in the old backup files, there is no subsequence number, as each version’s mutations are serialized in order in one file.
+* [PR #11901](https://github.com/apple/foundationdb/pull/11901) adds `RestoreDispatchPartitionedTaskFunc` to support restoring from partitioned-log-format backup. At a high level, backup agents will read partitioned mutation logs from the blob store, convert them into the old mutation log format in memory, and write mutations to the target database. All the rest of restore process is the same as before.
 
-### Integration with the [Performant Restore System](https://github.com/apple/foundationdb/issues/1049)
+  Specifically, `StartFullRestoreTaskFunc` kicks off the process by setting up some variable and start the first `RestoreDispatchPartitionedTaskFunc`, and each `RestoreDispatchPartitionedTaskFunc` with an input range `[v0, v1)` is to orchestrate the restore of both log and range file, then start the next `RestoreDispatchPartitionedTaskFunc` with an input range `[v1, v2)`, or finish the process if `v1 > targetVersion`.
+
+  There are many `RestoreRangeTaskFunc` for a single `RestoreDispatchPartitionedTaskFunc`, each responsible to restore a block of file, and one `RestoreLogDataPartitionedTaskFunc` for a single `RestoreDispatchPartitionedTaskFunc`. `RestoreRangeTaskFunc` would restore the keys to their original key while `RestoreLogDataPartitionedTaskFunc` would restore the keys to an intermediate state with a certain prefix(`alog`) who can be recognized by the commit proxy later to convert them back from this intermediate format into their original key value pairs. This is to make sure smaller version mutations do not override later version range file keys.
```

Comment:
Just for my understanding, what does 'block of file' mean? Is it some fixed size/range in the file.

nit:
who can be recognized -> may be 'which can be recognized ' sounds better?